From 6f10a741433104b229361fb3c8405f7556e07378 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 23 Dec 2017 23:00:46 +0200
Subject: [PATCH 143/249] lineage: Remove libtruezip

* This is a java library that was used in CMFileManager
* It's no longer being used or compiled, so kill it.

Change-Id: I9bc6f310c415bef029471edb9345f9d0bee80c50
---
 snippets/lineage.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index a67b1de..376ad02 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -13,7 +13,6 @@
   <project path="external/libncurses" name="LineageOS/android_external_libncurses" />
   <project path="external/libnfc-nxp" name="LineageOS/android_external_libnfc-nxp" />
   <project path="external/libtar" name="LineageOS/android_external_libtar" />
-  <!--<project path="external/libtruezip" name="LineageOS/android_external_libtruezip" />-->
   <project path="external/lineage-sdk-api-coverage" name="LineageOS/android_external_lineage-sdk-api-coverage" />
   <!--<project path="external/lzo" name="LineageOS/android_external_lzo" />-->
   <project path="external/nano" name="LineageOS/android_external_nano" />
-- 
2.17.1

